integration-tests/tests/src/tests/sync/app.ts

```diff
@@ -235,6 +236,50 @@ describe("App", () => {
       expect(this.app.currentUser).to.be.null;
     });
 
+    it("currentUser is available from an App listener", async function (this: Mocha.Context & AppContext & RealmContext) {
+      expect(this.app.currentUser).to.be.null;
+
+      const credentials = Realm.Credentials.anonymous();
+
+      const handle = createPromiseHandle();
+      const listener = () => {
+        expect(this.app.currentUser).to.not.be.null;
+        this.app.removeListener(listener);
+        handle.resolve();
+      };
+
+      this.app.addListener(listener);
+      await this.app.logIn(credentials);
+
+      await this.app.currentUser?.logOut();
+      expect(this.app.currentUser).to.be.null;
+
+      await handle;
+    });
+
+    it.only("currentUser is available from a User listener", async function (this: Mocha.Context & AppContext & RealmContext) {
+      expect(this.app.currentUser).to.be.null;
+
+      const credentials = Realm.Credentials.anonymous(false);
```

Comment:
Will the `false` actually make a difference here? Or can it not be guaranteed that this is the only logged in user when this test is run? (The previous test used the default argument.)

Reply:
I agree. It'll most likely not make a difference.